index.html

```diff
@@ -14,18 +14,23 @@
         <nav class="nav-list">
             
             <a class="logo" href="#">My Logo</a>
+            <ul class="navbar-list">
 
-            <button class="menu-button">
+            <li class="menu-icon">
                 <img class="menu" src="static/images/Union.png" alt=" navigation" />
-            </button>
+            </li>
+            <li class="desktop-nav-list">Portfolio</li>
+            <li class="desktop-nav-list">About</li>
+            <li class="desktop-nav-list">Contact</li>
+          </ul>
 
         </nav>
 
 
     </header>
 
     <section class="container">
-        <h1 class="greeting-text">I am Leslie Gudo <br />Glad to see you!</h1>
+        <h1 class="greeting-text">I am Leslie Glad to see you!</h1>
```

Comment:
- [ ] KIndly break the content of this headline text to match the figma design 👍🏽 

Figma design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/213546548-5c6e516c-676d-440a-a6dc-c62d1f3ed193.png)

Your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/213546670-0a25f94e-c8d9-4c79-a4c6-5594ea72a452.png)